src/coreclr/debug/daccess/enummem.cpp

```diff
@@ -318,8 +318,12 @@ HRESULT ClrDataAccess::EnumMemDumpJitManagerInfo(IN CLRDataEnumMemoryFlags flags
 
     if (flags == CLRDATA_ENUM_MEM_HEAP2)
     {
-        EEJitManager* managerPtr = ExecutionManager::GetEEJitManager();
+        EECodeGenManager* managerPtr = ExecutionManager::GetEEJitManager();
         managerPtr->EnumMemoryRegions(flags);
+#ifdef FEATURE_INTERPRETER
+        managerPtr = ExecutionManager::GetInterpreterJitManager();
```

Comment:
Should it be using `#ifdef` `#else` blocks (to avoid overwriting the values)?

Reply:
It should enumerate both memory regions and it just reuses local variable. This seems fine.

Reply:
Ah, ok if enumerating both is the intention, it's fine.

Reply:
No, it needs to call the enumeration for both managers